web/display/minimal/gaugevalues.php

```diff
@@ -1,22 +1,19 @@
-<?php
-
-$result = '';
-$lines = file($_SERVER['DOCUMENT_ROOT'].'/openWB/openwb.conf');
-foreach($lines as $line) {
- 	if(strpos($line, "displayaktiv=") !== false) {
-		list(, $displayaktivold) = explode("=", $line);
-	}
-	if(strpos($line, "displaylp1max=") !== false) {
-		list(, $displaylp1maxold) = explode("=", $line);
-	}
-	// if(strpos($line, "displaylp2max=") !== false) {
-	// 	list(, $displaylp2maxold) = explode("=", $line);
-	// }
-	if(strpos($line, "rfidakt=") !== false) {
-		list(, $rfidaktold) = explode("=", $line);
-	}
-	if(strpos($line, "isss=") !== false) {
-		list(, $isssold) = explode("=", $line);
-	}
-}
-?>
+<?php
+
+$result = '';
+$lines = file($_SERVER['DOCUMENT_ROOT'].'/openWB/openwb.conf');
+foreach($lines as $line) {
```

Comment:
Der alte Code hat so seine Tücken. Beispielsweise werden auch die Zeilenumbrüche am Ende mit in die Variable geschrieben. In anderen Bereichen (Themes, Einstellungen) wird inzwischen folgende Variante zum Einlesen der openwb.conf genutzt:
```
$lines = file($_SERVER['DOCUMENT_ROOT'] . '/openWB/openwb.conf');
foreach($lines as $line) {
	list($key, $value) = explode("=", $line, 2);
	${$key."old"} = trim( $value, " '\t\n\r\0\x0B" ); // remove all garbage and single quotes
}
```
Bitte die `foreach` Schleife entsprechend anpassen und in weiteren Dateien nachsehen, ob auf einen Wert mit "\n" geprüft wird.

Reply:
Ich denke dafür brauchen wir kein separates file, habs nach index.php verschoben

Reply:
Klar, geht auch.